[Bug] test_rock_agent_run_iflow fix

**Bug Description**
test_rock_agent_run_iflow not pass
FAILED tests/integration/sdk/sandbox/agent/rock_agent/test_run.py::test_rock_agent_run_iflow - AssertionError: assert 'Hello! I am ROCK' in 'Auth method has been deprecated. Please reconfigure with OpenAI Compatible API.'
**Steps to Reproduce**

**Expected Behavior**

**Actual Behavior**

**Error Logs**

**Environment Information**
- **OS**: [e.g. Ubuntu 22.04, macOS 13.0, Windows 11]
- **Python Version**: [e.g. 3.11.5]
- **ROCK Version**: [e.g. 0.2.0]
- **Installation Method**: [e.g. pip install rl-rock, source installation with uv]
- **Docker Version**: [e.g. 24.0.6] (if using sandbox features)
- **Deployment Type**: [e.g. local, distributed, ray]

**ROCK Configuration**
- **Runtime Environment Type**: [e.g. uv, pip, conda]
- **Sandbox Image**: [e.g. python:3.11, custom image]
- **Resource Allocation**: [e.g. memory=8g, cpus=2.0]

**Component Affected**
- [ ] Sandbox
- [ ] Actions
- [ ] Deployments
- [x] SDK & API
- [ ] Envhub
- [ ] CLI 
- [ ] Performance & Optimization
- [ ] Documentation & Examples

--- a/tests/integration/sdk/sandbox/agent/rock_agent/iflow_config.yaml
+++ b/tests/integration/sdk/sandbox/agent/rock_agent/iflow_config.yaml
@@ -2,7 +2,7 @@ run_cmd: "iflow -p ${prompt} --yolo"
 
 runtime_env_config:
   type: node
-  custom_install_cmd: "npm i -g @iflow-ai/iflow-cli@latest"
+  custom_install_cmd: "npm i -g @iflow-ai/iflow-cli@0.5.17"
 
 env:
   IFLOW_API_KEY: "xxxxxxx"

